spec/support/simplecov.rb

```diff
@@ -1,5 +1,6 @@
 require 'simplecov'
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([SimpleCov::Formatter::HTMLFormatter])
+require 'simplecov-json'
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([SimpleCov::Formatter::HTMLFormatter, SimpleCov::Formatter::JSONFormatter])
```

Comment:
Should we split them into 2 lines?
```suggestion
formatters = [SimpleCov::Formatter::HTMLFormatter, SimpleCov::Formatter::JSONFormatter]

SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(formatters)
```

Reply:
👍   @hoangmirs. Done in [4fc2d0d](https://github.com/nimblehq/rails-templates/pull/229/commits/4fc2d0dd6a5ba2eba730af18ae6047c0e3008a48)